### PR TITLE
feat: add field-id field to form editor

### PR DESCRIPTION
**Pull request by opqdonut**
for #1804

# Checklist for author

Remove items that aren't applicable, check items that are done.

## Reviewability
- [x] Link to issue

## Documentation
- [x] Update changelog if necessary
- [x] Update manual/ (if applicable)

## Testing
- [ ] Complex logic is unit tested
- [x] Valuable features are integration / browser / acceptance tested automatically

### CODE REVIEW

File: resources/translations/en.edn [hunk 1]
```diff
@@ -266,6 +266,7 @@
                 :column-n "Column %1"
                 :column-key "ID code"
                 :column-label "Label"
+                :field-id "Field ID code"
```
Comment (Macroz) [inline resources/translations/en.edn:269]: Maybe shorten to just field id?

File: src/cljs/rems/administration/create_form.cljs [hunk 2]
```diff
@@ -281,6 +281,10 @@
   [localized-text-field context {:keys [:form/external-title]
                                  :label (text :t.administration/external-title)}])
 
+(defn- form-field-id-field [field-index]
+  [text-field context {:keys [:form/fields field-index :field/id]
+                       :label (text :t.create-form/field-id)}]) ; TODO localize
```
Comment (Macroz) [inline src/cljs/rems/administration/create_form.cljs:286]: Isn't this localized now?

**Review by Macroz — APPROVED** [review]

File: resources/translations/en.edn (hunk 1 above)
Comment (opqdonut) [reply resources/translations/en.edn:269]: I was trying to mirror the "ID code" we use for options & table columns

**Comment by opqdonut** [comment]
following up with some browser tests

File: resources/translations/en.edn (hunk 1 above)
Comment (Macroz) [reply resources/translations/en.edn:269]: Well the Swedish translation does not follow the same "code"? And it seems overly complicated (perhaps it was not really thought about for the options either).

Comment (opqdonut) [reply resources/translations/en.edn:269]: Yeah for swedish I was mirroring the `:option-key "Id"` translation that was already there. I'm fine with dropping "code" from all the english localizations.

**Review by marharyta — APPROVED** [review]

**Pull request merged**